service/s3/s3manager/upload_test.go

```diff
@@ -272,7 +272,7 @@ func TestUploadFailIfPartSizeTooSmall(t *testing.T) {
 		t.Errorf("Expected %q, but received %q", "ConfigError", aerr.Code())
 	}
 
-	if strings.Contains("part size must be at least", aerr.Message()) {
+	if strings.Contains(aerr.Message(), "part size must be at least") {
```

Comment:
Looks like this assert was broken, and failing on the wrong thing. think this code block would be better suited as the following. I think this change will also fix the broken unit tests in CR for this PR.

```go
if e, a := "part size must be at least", aerr.Message(); !strings.Contains(a, e) {
    t.Errorf("expect %v to be in %v", e, a)
}
```

This style also brings this assert inline with the SDK's preferred assert test pattern.